packages/flutter/test/foundation/_with_flutter_leak_tracking.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Why the `_` in the file name? Can we just rename that to `leak_tracking.dart`? Similarly to how we have `packages/flutter/test/widgets/semantics_tester.dart`, `packages/flutter/test/rendering/mock_canvas.dart` and other internal test utils?